qtpy/__init__.py

```diff
@@ -207,17 +211,26 @@ class PythonQtWarning(Warning):
 def _warn_old_minor_version(name, old_version, min_version):
     """Warn if using a Qt or binding version no longer supported by QtPy."""
     warning_message = (
-        "{name} version {old_version} is no longer supported upstream or "
+        "{name} version {old_version} is not supported upstream or "
         "by QtPy 2.0. To ensure your application works correctly with QtPy, "
```

Comment:
Maybe we can simplify the warning message like this?
```suggestion
        "{name} version {old_version} is not supported by QtPy. "
        "To ensure your application works correctly with QtPy, "
```

Reply:
Agreed; I almost changed to something resembling that, but just didn't want to stray too far from the message we'd agreed on previously.